spec/src/modules/tracker.js

```diff
@@ -24,6 +24,7 @@ const bundled = process.env.BUNDLED === 'true';
 const skipNetworkTimeoutTests = process.env.SKIP_NETWORK_TIMEOUT_TESTS === 'true';
 const bundledDescriptionSuffix = bundled ? ' - Bundled' : '';
 const timeoutRejectionMessage = bundled ? 'AbortError: Aborted' : 'AbortError: The user aborted a request.';
+const testAnalyticsTag = { param1: 'test', param2: 'test2' };
```

Comment:
Github won't let me add a comment to the correct line but can we add the `analyticsTags` to the trackInpuFocusV2 event on line 400